pegasus/helpers/hourofcode_helpers.rb

```diff
@@ -152,23 +152,30 @@ def localized_image(path)
 
 def campaign_date(format)
   @country ||= hoc_detect_country
+  type = HOC_COUNTRIES[@country]['type'] || 'default'
+  language = HOC_COUNTRIES[@country]['default_language']
+  id = 'campaign_date_full'
 
   case format
   when "start-short"
-    return HOC_COUNTRIES[@country]['campaign_date_start_short']
+    id = 'campaign_date_start_short'
   when "start-long"
-    return HOC_COUNTRIES[@country]['campaign_date_start_long']
+    id = 'campaign_date_start_long'
   when "short"
-    return HOC_COUNTRIES[@country]['campaign_date_short']
+    id = 'campaign_date_short'
   when "full"
-    return HOC_COUNTRIES[@country]['campaign_date_full']
+    id = 'campaign_date_full'
   when "year"
-    return HOC_COUNTRIES[@country]['campaign_date_year']
+    id = 'campaign_date_year'
   when "full-year"
-    return HOC_COUNTRIES[@country]['campaign_date_full_year']
-  else
-    return HOC_COUNTRIES[@country]['campaign_date_full']
+    id = 'campaign_date_full_year'
+  end
+
+  if %w(latam europe africa).include? type
```

Comment:
Is it worth doing the opposite of this? i.e. `unless type == 'default'`. I feel like it's possible that we add another set of dates and then we would only have to edit `countries.json` and `en.yml`

Reply:
that's fair, I was worried about the likelihood of someone putting an incorrect type in there, in which case we would want to fall back to default instead of using an id that doesn't exist

Reply:
Ooo good point!